{"name": "test_add_error[Ca_Add_013]", "status": "passed", "parameters": [{"name": "numa", "value": "4"}, {"name": "numb", "value": "'å­—'"}, {"name": "conclusion", "value": "'TypeError'"}], "start": 1648453312609, "stop": 1648453312609, "uuid": "ec7c9f34-5b3f-42cb-a7e1-f060843facdc", "historyId": "66ce6f79dedfee2848ff1a32bfb7ba9d", "testCaseId": "a3bfe621045782c9ea9a5de99829cd04", "fullName": "test_pytest.test_cal.TestCal#test_add_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('def_fixture')"}, {"name": "parentSuite", "value": "test_pytest"}, {"name": "suite", "value": "test_cal"}, {"name": "subSuite", "value": "TestCal"}, {"name": "host", "value": "IT-L-YANGYL"}, {"name": "thread", "value": "3432-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest.test_cal"}]}